### PR TITLE
instruction for make command on windows

### PR DESCRIPTION
**Pull request by prashant-isbest**
Signed-off-by: Prashant Joshi <scienceprashant27@gmail.com>

**Description**

This PR fixes # contributing.md

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ Yes] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 89cca95c34f52785f280f8b0f2d30d7979047f4a at: https://61517c2eda3c8a72e667133e--layer5ng.netlify.app

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -87,7 +87,15 @@ Please contribute! The Layer5 site uses Gatsby. The process of contributing to d
 `vi <specific page>.md` # or use your favorite IDE
 1. Run site locally to preview changes.
 `make site` # this will run a local web server with "live reload" conveniently enabled.
-(NOTE: while using `make` command on windows, there sometimes arises an error in identifying the command even after it is installed(unrecognized command), this is because the PATH for the binary might not be set correctly.)
+     
+    Installing make on windows using choco
+    1. open powershell as administator
+    2. Copy the below command and run it
+    3. `Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))`
```
Comment (Nikhil-Ladha) [inline CONTRIBUTING.md:94]: Do you know what this does?

Comment (Nikhil-Ladha) [reply CONTRIBUTING.md:94]: Then, prefer to always explain what changes you are doing.
And, add the justification of using the command as a comment in beside it.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -87,7 +87,15 @@ Please contribute! The Layer5 site uses Gatsby. The process of contributing to d
 `vi <specific page>.md` # or use your favorite IDE
 1. Run site locally to preview changes.
 `make site` # this will run a local web server with "live reload" conveniently enabled.
-(NOTE: while using `make` command on windows, there sometimes arises an error in identifying the command even after it is installed(unrecognized command), this is because the PATH for the binary might not be set correctly.)
+
```
Comment (Nikhil-Ladha) [inline CONTRIBUTING.md:90]: Also, add a note to follow this process only in case one faces an issue with the make command.

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -87,7 +87,15 @@ Please contribute! The Layer5 site uses Gatsby. The process of contributing to d
 `vi <specific page>.md` # or use your favorite IDE
 1. Run site locally to preview changes.
 `make site` # this will run a local web server with "live reload" conveniently enabled.
-(NOTE: while using `make` command on windows, there sometimes arises an error in identifying the command even after it is installed(unrecognized command), this is because the PATH for the binary might not be set correctly.)
```
Comment (warunicorn19) [inline CONTRIBUTING.md:90]: Also, there's no need to delete this note, we could keep it here.

**Pull request closed without merging**